test_sets/rc_OnRemoteControlSettings.txt

```diff
@@ -1,6 +1,14 @@
+./test_scripts/RC/OnRemoteControlSettings/001_SUCCESS_in_case_AUTO_ALLOW_seq_1.lua
+./test_scripts/RC/OnRemoteControlSettings/002_SUCCESS_in_case_AUTO_ALLOW_seq_2.lua
+./test_scripts/RC/OnRemoteControlSettings/003_SUCCESS_in_case_AUTO_ALLOW_seq_3.lua
+./test_scripts/RC/OnRemoteControlSettings/004_SUCCESS_in_case_AUTO_ALLOW_seq_4.lua
+./test_scripts/RC/OnRemoteControlSettings/005_REJECTED_in_case_nonFULL_HMI_level.lua
+./test_scripts/RC/OnRemoteControlSettings/006_IN_USE_in_case_2_requests.lua
+./test_scripts/RC/OnRemoteControlSettings/007_IN_USE_in_case_AUTO_DENY.lua
 ./test_scripts/RC/OnRemoteControlSettings/008_Allowed_false.lua
 ./test_scripts/RC/OnRemoteControlSettings/009_Allowed_true_accessMode_AUTO_ALLOW.lua
 ./test_scripts/RC/OnRemoteControlSettings/010_Allowed_true_accessMode_AUTO_DENY.lua
 ./test_scripts/RC/OnRemoteControlSettings/011_Allowed_true_accessMode_ASK_DRIVER.lua
 ./test_scripts/RC/OnRemoteControlSettings/012_Default_accessMode.lua
 ./test_scripts/RC/OnRemoteControlSettings/013_sequence_switches_of_accessMode.lua
+./test_scripts/RC/OnRemoteControlSettings/014_Invalid_data_from_HMI.lua
```

Comment:
@dboltovskyi Please add to current test set new test 
(./test_scripts/RC/OnRemoteControlSettings/015_Empty_data_from_HMI.lua)